README.md

```diff
@@ -57,54 +45,49 @@ Then, navigate to the testnet folder, like so:
 cd $GOPATH/src/github.com/dusk-network/dusk-blockchain
 ```
 
-Then, to build the binary, simply run:
+To build the binary, simply run:
 
 ```bash
 make build
 ```
 
-Next up, we build the rust part of the node. Instructions for building the `blindbid` module can be found [here](https://github.com/dusk-network/dusk-blindbidproof/blob/master/Readme.md#how-to-build). After building, make sure you move the executable into the `testnet` folder.
-
 Finally, to start your node, type:
 
 ```bash
 ./bin/dusk --config=dusk.toml
 ```
 
-## Features
+## Wallet
 
-1. Cryptography Module - Includes an implementation of SHA-3 and LongsightL hash functions, Ristretto and BN-256 elliptic curves, Ed25519, BLS, bLSAG and MLSAG signature schemes, Bulletproofs zero-knowledge proof scheme.
-2. Consensus Module - Includes a complete implementation of the latest version of Segregated Byzantine Agreement \(v2.1\) consensus protocol, which contains three phases - Block Generation, Block Reduction and Block Agreement as well as the Blind Bid proof protocol utilized in the Block Generation phase.
-3. Networking and Database Module - Includes the blockchain storage and related logic, as well as the implementation of P2P gossip protocol utilized by the Dusk Network protocol.
-4. CLI Wallet - Includes an functionality enabling the user to create/load a wallet, transfer/bid/stake DUSK tokens, and managing the node. 
+The wallet is hosted in a separate folder, [found here](https://github.com/dusk-network/cmd/wallet). 
```

Comment:
This link is broken. I think you forgot `dusk-blockchain` in the URL. However, currently there is no README in the `wallet/` folder

Reply:
Good catch, will fix it